pkg/cmd/target.go

```diff
@@ -272,8 +273,13 @@ func NewTargetCmd(targetReader TargetReader, targetWriter TargetWriter, configRe
 						checkError(err)
 					}
 				}
+			}
 
+			err := historyWriter.Write(pathHistory, strings.Join(os.Args[0:], " "))
```

Comment:
do you consider the complete target stack?

```bash
g target garden g1
g target project p1
g target shoot s1
g target project p2
g target shoot s2
g x # select the targeting of shoot s1 from the history
```

Reply:
so far not in the scope.  the history only records/persent what user use `g target` command. no black box when the user recall targeting history from the history list